packages/cli/src/tests/utils/build.js

```diff
@@ -148,6 +148,58 @@ describe('build', () => {
       });
   });
 
+  it('should make a build.zip without .zapierapprc', () => {
```

Comment:
Can you explain why this test is needed?

Reply:
I changed the build command to no longer rely on there being a `.zapierapprc`. Users probably won't notice, but I ran into it a lot when testing build sizes - it's nice not to have to register the app (or use a fake file). 

I later found a bug where build would fail without an `.zapierapprc`, so I fixed it and added a test.